pkg/velerorunner/task.go

```diff
@@ -43,6 +43,7 @@ type Task struct {
 	Phase           string
 	Backup          *velero.Backup
 	Restore         *velero.Restore
+	Annotations     map[string]string
```

Comment:
Please document the `Annotations` field.

Also, (I doubt this was obvious and me being OCD but ..) I have been trying to group the required fields passed in on construction together and putting the _transient_ fields at the end.

Reply:
I did not realize you were doing this, good to know.